tests/collector/integration.js

```diff
@@ -0,0 +1,957 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/collector/integration.js
+ *
+ * This is a simulation of a full collector setup: one refocus instance
+ * and multiple separate collectors. The collector runs in a separate process,
+ * time is mocked, and requests are intercepted.
+ *
+ * To run this, you will need to have the collector source code in the same directory
```

Comment:
If we publish refocus-collector as npm module, would we then be able to make that a *dev* dependency and execute these integration tests?

Reply:
Yeah, that should work, and it would allow us to set it up to run in travis.
It would mean we would have to pulish to npm any time something changes that could affect the tests.

Reply:
published and updated.

Reply:
Confirmed that it runs in travis now, but I'm disabling it for now because there are still flappers that need to be cleaned up in a future story.